spec/views/hyrax/dashboard/profiles/edit.html.erb_spec.rb

```diff
@@ -38,16 +38,27 @@
   end
 
   context "with trophy" do
-    let(:solr_document) { SolrDocument.new(id: 'abc123', has_model_ssim: 'GenericWork', title_tesim: ['Title']) }
     let(:page) { Capybara::Node::Simple.new(rendered) }
 
-    before do
-      assign(:trophies, [Hyrax::TrophyPresenter.new(solr_document)])
-      render
+    context "and there are no highlighted works" do
```

Comment:
as this logic is in the `_trophy_edit.html.erb` partial, I think it would make sense to create a test `spec/views/hyrax/dashboard/profiles/_trophy_edit.html.erb_spec.rb` and put these tests there.

Reply:
It sounds good and those tests are moved out to spec/views/hyrax/dashboard/profiles/_trophy_edit.html.erb_spec.rb: https://github.com/samvera/hyrax/pull/2732/commits/9909f44b1a755bca2be9c1bc51ecc9bfa7cf7afb.